WordPress/pattern-directory
# Add/pattern grid loading

This PR adds a skeleton card to display while we are loading patterns. The implementation differs slightly from the designs seeing that meta-information has drifted.

This PR doesn't address 2 outstanding issues:

1. All the `iframe`s load at once and are messy when rendering. We should do something about that in the following PR
2. The context bar loading seems to be fairly buggy. Its height is inaccurate. 

Designs: #101 

### Screenshots

![](https://d.pr/i/zOIhns.png)

Check out the [animated GIF.](https://d.pr/i/PQ453A)



### How to test the changes in this Pull Request:

1. Visit `/`
2. Expect to see a loading state
3. Click on a category, expect to see the loading state
